src/test/java/hudson/plugins/active_directory/ActiveDirectoryDomainIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+package hudson.plugins.active_directory;
+
+import org.htmlunit.FailingHttpStatusCodeException;
+import org.htmlunit.html.HtmlButton;
+import org.htmlunit.html.HtmlElement;
+import org.htmlunit.html.HtmlForm;
+import org.htmlunit.html.HtmlInput;
+import org.htmlunit.html.HtmlPage;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
+import static hudson.Functions.isWindows;
+
+public class ActiveDirectoryDomainIntegrationTest {
+	@Rule
+	public JenkinsRule jenkins = new JenkinsRule();
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@ClassRule
+	public static FlagRule<String> fipsSystemPropertyRule =
+			FlagRule.systemProperty("jenkins.security.FIPS140.COMPLIANCE", "true");
+
+
+	/**
+	 * Tests the behavior of the "Save" button when a short password is configured.
+	 *
+	 * <p>For the preconfigured value, the password is "small" in the local data.
```

Comment:
So this test would break when adding the readResolve check.

Reply:
@fcojfernandez  had pushed some changes that fixed this.
I believe this can be marked as removed